app/src/main/java/com/duckduckgo/app/sitepermissions/permissionsperwebsite/WebsitePermissionSettingOption.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.sitepermissions.permissionsperwebsite
+
+import androidx.annotation.StringRes
+import com.duckduckgo.app.browser.R
+import com.duckduckgo.site.permissions.store.sitepermissions.SitePermissionAskSettingType
+import java.io.Serializable
+
+data class WebsitePermissionSetting(
+    val icon: Int,
+    val title: Int,
+    val setting: WebsitePermissionSettingOption,
+) : Serializable
+
+enum class WebsitePermissionSettingOption(
+    val order: Int,
+    @StringRes val stringRes: Int,
+) {
+    ASK(1, R.string.permissionsPerWebsiteAskSetting),
```

Comment:
should this be a 0? weird to have a duplicated order

Reply:
Talked about it. All fine 👍